pkg/reconciler/route/route.go

```diff
@@ -288,13 +301,30 @@ func (c *Reconciler) reconcile(ctx context.Context, r *v1alpha1.Route) error {
 		Hostname: resourcenames.K8sServiceFullname(r),
 	}
 
+	tls := []netv1alpha1.ClusterIngressTLS{}
+	if config.FromContext(ctx).Network.AutoTLS {
+		domains := tr.GetDomains(r.Status.Domain, traffic.Targets)
+		// TODO(zhiminx): add a feature flag in config-network ConfgiMap to control the types of certs (wildcard/non-wildcard)
+		desiredCerts := resources.MakeCertificates(r, domains, true)
```

Comment:
The bool is unclear, consider a Go enum instead (look for `iota`)

Reply:
as we are focusing on non-wildcard cert, we no longer need this bool now.